IFC 4.3/Alignment_3/UT_Alignment_3.ifc

```diff
@@ -0,0 +1,87 @@
+#8= IFCAPPLICATION(#9,'0.1.6.0','GeometryGymIFC v0.1.6.0','GeometryGymIFC v0.1.6.0');
+#9= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#10= IFCOWNERHISTORY(#7,#8,$,.ADDED.,1608552399,$,$,1608552399);
+#11= IFCPROJECT('0HbUe88sjA4x1itaa_HkeP',#10,'sampleProject',$,$,$,$,(#38),#12);
+#12= IFCUNITASSIGNMENT((#13,#14,#15,#16,#17));
+#13= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#14= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#18= IFCRELAGGREGATES('09HZaegnjAoPpjN_4YPw9K',#10,$,$,#11,(#4));
+#19= IFCCARTESIANPOINT((0.0,0.0));
+#20= IFCCARTESIANPOINT((0.0,0.0));
+#21= IFCAXIS2PLACEMENT2D(#20,$);
+#22= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#23= IFCALIGNMENT('2gbIbH1Vj7rBXLBY$zNrRB',#10,'DefaultAlignment',$,$,#26,$,$,$);
+#24= IFCRELCONTAINEDINSPATIALSTRUCTURE('3Mdlw$O9PFK9KZrLjjJ4Fg',#10,'Site','Site Container for Elements',(#23),#4);
+#25= IFCAXIS2PLACEMENT3D(#22,$,$);
+#26= IFCLOCALPLACEMENT($,#25);
+#27= IFCCARTESIANPOINT((-29.29264,-6.90761));
+#28= IFCALIGNMENTHORIZONTALSEGMENT($,$,#27,0.462753315190078,0.0,0.0,72.3546021022319,$,.LINE.);
+#29= IFCLOCALPLACEMENT($,#21);
+#30= IFCALIGNMENTHORIZONTAL('0PWroiQY1Ag9ut1woZ1bKx',#10,$,$,$,#29,#44,0.0,(#28));
+#31= IFCAXIS2PLACEMENT2D(#27,#32);
+#32= IFCDIRECTION((0.89482677,0.44641354));
+#33= IFCDIRECTION((1.0,0.0));
+#34= IFCVECTOR(#33,1.0);
+#35= IFCLINE(#19,#34);
+#36= IFCCURVESEGMENT(.DISCONTINUOUS.,#31,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#35);
+#37= IFCCOMPOSITECURVE((#36),.F.);
+#38= IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,0.0001,#40,#41);
+#39= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#40= IFCAXIS2PLACEMENT3D(#39,$,$);
+#41= IFCDIRECTION((0.0,1.0));
+#42= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#38,$,.GRAPH_VIEW.,$);
+#43= IFCSHAPEREPRESENTATION(#42,'Axis','Curve2D',(#37));
+#44= IFCPRODUCTDEFINITIONSHAPE($,$,(#43));
+#45= IFCRELAGGREGATES('2Jyg9f5THBhPuqj7M9RBXD',#10,$,$,#23,(#30));
+#46= IFCALIGNMENTSEGMENT('1a97S0Thb0dRIuIHtheVwB',#10,$,$,$,#29,#49,#28);
+#47= IFCRELNESTS('15V22QdifE_Qri3Uh1CmAK',#10,$,$,#30,(#46));
+#48= IFCSHAPEREPRESENTATION(#42,'Axis','Curve2D',(#36));
+#49= IFCPRODUCTDEFINITIONSHAPE($,$,(#48));
+#50= IFCALIGNMENTVERTICALSEGMENT($,$,0.0,72.349260880448,11.0,-0.0121508006435841,-0.0121508006435841,$,.CONSTANTGRADIENT.);
+#51= IFCDIRECTION((0.99992618,-0.0121505));
+#52= IFCVECTOR(#51,72.3546021022319);
+#53= IFCCARTESIANPOINT((0.0,0.0));
+#54= IFCLINE(#53,#52);
+#55= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(0.0),$,$,$,#37);
+#56= IFCAXIS2PLACEMENTLINEAR(#55,$,$);
+#57= IFCCURVESEGMENT(.DISCONTINUOUS.,#56,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#54);
+#58= IFCGRADIENTCURVE(#37,(#57),$);
+#59= IFCLOCALPLACEMENT($,#21);
+#60= IFCALIGNMENTVERTICAL('1setzPyrX1cQv3iei$benF',#10,$,$,$,#59,#67,(#50));
```

Comment:
I can't identify an aggregation relationship between alignment entity and vertical alignment entity.

Reply:
Connection added in 37b6312